sites/mosul/pom.xml

```diff
@@ -140,8 +140,6 @@
                 <echo message="Adding MSF Mosul frontend config"/>
                 <replaceregexp file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/.env"
                             match="(SPA_CONFIG_URLS=.+)" replace="\1,/openmrs/spa/configs/msf-mosul-frontend-config.json" />
-                <replaceregexp file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/concatenated.env"
```

Comment:
Was this initially added as part of the override?

Reply:
This is was a duplicate.

On building, the `.env` file is given priority.  Its contents are merged with the existing `.env` files to generate the `concatenated.env`.  
However, at runtime, the `concatenated.env` file is given priority which means that all our modifications at compile-time should be in the `.env` and modifications at runtime should be in the `concatenated.env`.
For example, changing the OpenMRS port on UAT or DEV should be done in the `concatenated.env` file.